app/api/cloudinary/route.ts

```diff
@@ -0,0 +1,18 @@
+// pages/api/upload.js
+import cloudinary from '../../../utils/cloudinary';
```

Comment:
remove comment,
and use alias to import instead relative.
like: `@/utils/cloudinary`